SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex

```diff
@@ -2464,13 +2478,16 @@ \subsection{Alignment}
 
 \small
 \begin{minted}[baselinestretch=1,fontsize=\footnotesize,linenos=false,bgcolor=ltgray]{cpp}
-InputPixelType  m_ForegroundValue;
-OutputPixelType m_BackgroundValue;
+bool                        m_Quiet{ false };
+bool                        m_TestAbort{ false };
+std::string                 m_Comment;
+itk::ProcessObject::Pointer m_Process;
 
-unsigned int m_MaximumIterations;
+CommandType::Pointer m_StartFilterCommand;
+CommandType::Pointer m_EndFilterCommand;
```

Comment:
Perhaps add a comment that smart pointers are automatically initialized to `nullptr`? Or maybe do that for line 2484, and change the type here to something else?

Reply:
> Perhaps add a comment that smart pointers are automatically initialized to nullptr

I did not do it because smart pointer automatic initialization should have been mentioned before in some other section.

Resolving since I believe 55ea845 now demonstrates the point well.